test/Controllers/UsersTest.php

```diff
@@ -69,80 +69,101 @@ public function testCreateUser() {
      */
-    public function testUpdateUserById() {
-        // // $this->expectException(EntitiesErrorException::class);
-        //
-        // $id = 'id';
-        // $updateUserByIdBody = new UpdateUserByIdBody();
-        // $response = $this->users->updateUserById(
-        //         $this->authorization,
-        //         $id,
-        //         $updateUserByIdBody);
-        // print_r($response);
-
-        // $this->assertObjectHasAttribute('user', $response);
-        // $this->assertEquals($id, $response->user->accountManagerId);
+    public function testUpdateUserById() { // $this->expectException(EntitiesErrorException::class);
+        // Create user:
+        $createUserBody = $this->uniqueUserData();
+        $createResponse = $this->users->createUser($this->authorization, $createUserBody);
+        // /* DEBUG */ print_r($createResponse);
+
+        $anotherUserBody = $this->uniqueUserData();
+
+        // Update user by it's id
+        $response = $this->users->updateUserById(
+            $this->authorization,
+            $createResponse->user->id,
+            $anotherUserBody);
+        // /* DEBUG */ print_r($response);
+
+        // Was it fetched?
+        $this->assertObjectHasAttribute('user', $response);
+
+        // Does it have all the new data we sent?
+        $responseBody = $this->dataFromResponse($response);
+        $this->assertEquals($responseBody, $anotherUserBody);
+
+        // XXX Do we need to getUserById again and/or check the ids?
     }
 
     /**
      * Test GET /users
      * GonebusyLib\Controllers\UsersController::getUsers()
      */
     public function testGetUsers() {
-
+        $perPage = 3; // XXX Are this enough?
         $response = $this->users->getUsers(
             $this->authorization,
             $page = 1,
-            $perPage = 3);
-        // print_r($response);
+            $perPage);
+        // /* DEBUG */ print_r($response);
 
+        // Did it return an arrayof 3 users?
         $this->assertObjectHasAttribute('users', $response);
-
-        // $testingUserBody = new CreateUserBody(
-        //     "8552697701",
-        //     "testing@gonebusy.com",
-        //     "8552697701",
-        //     "6815786092",
-        //     "pro"
-        // );
-        // print_r($testingUserBody);
-        //
-        // $responseBody = new CreateUserBody(
-        //     $response->users[0]->accountManagerId,
-        //     $response->users[0]->email,
-        //     $response->users[0]->id,
-        //     $response->users[0]->resourceId,
-        //     $response->users[0]->role
-        // );
-        // print_r($responseBody);
-        //
-        // $this->assertEquals($testingUserBody, $responseBody);
+        $this->assertCount($perPage, $response->users);
+        foreach($response->users as $user) {
+            $this->assertInstanceOf('GonebusyLib\Models\EntitiesUserResponse', $user);
+            // XXX Test structure?
```

Comment:
@alexagranov should we check the structure of all or some or one of the fetched users?

Thanks!